src/modules/auth/components/TermsComponent.tsx

```diff
@@ -0,0 +1,11 @@
+export function TermsComponent(): JSX.Element {
+  return (
+    <label className="label flex justify-start gap-2">
+      <input type="checkbox" className="checkbox" />
+      <span className="label-text">
+        Yes, I accept <span className="cursor-pointer">privice policy</span> &{' '}
```

Comment:
`span` => `a`
In the future, this will lead to an actual Privacy Policy page. 
Also, you have a typo here... To minimize typos, you can use: [Code Spell Checker](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker)